src/duckdb.js

```diff
@@ -215,6 +218,12 @@ async function insertArrowTable(database, name, table, options) {
   }
 }
 
+async function insertArqueroTable(database, name, source) {
+  const arrow = await loadArrow();
+  const table = arrow.tableFromIPC(source.toArrowBuffer());
+  return await insertArrowTable(database, name, table);
```

Comment:
I checked how Arquero is implemented, and _source_.toArrowBuffer calls _source_.toArrow internally.

https://github.com/uwdata/arquero/blob/1c8f438df033026535e1c4f67b4f891fa6f8f29e/src/table/column-table.js#L306-L308

https://github.com/uwdata/arquero/blob/1c8f438df033026535e1c4f67b4f891fa6f8f29e/src/format/to-arrow.js#L6-L8

So, this code is currently converting an Arquero Table to an Arrow Table to an Array Buffer then back again to an Arrow Table. You can skip two conversions by calling _source_.toArrow directly (and as we discussed earlier in this review, that means we should now check for the presence of _source_.toArrow instead of _source_.toArrowBuffer).

```suggestion
  return await insertArrowTable(database, name, source.toArrow());
```

Reply:
Thanks for taking a look, Mike! I started out trying to use _source_.toArrow rather than _source_.toArrowBuffer, but ran into errors that way (see screenshot), and the result of `requestDatabaseTables` for the cell when constructed that way is an empty array. 
![Screen Shot 2022-12-01 at 7 20 13 PM](https://user-images.githubusercontent.com/111310561/205207603-28c90e07-997a-4620-a4b2-83597d8b30ed.png)

Reply:
What version of Arquero are you using?

Our DuckDBClient depends on Apache Arrow version 9 or later, which means you’ll need to use Arquero version 5 or later. The version of Arquero in the Observable Standard library is currently 4.8.8 which uses Apache Arrow ^3.0.0.

So, you’ll need to load a more recent version of Arquero for this to work, and more generally, we need to implement standard library versioning so that we can upgrade Arquero to a more recent version in new notebooks.

Reply:
I'm using Arquero from our stdlib; my opinion is that this should work with the Arquero in our stdlib. I added a comment in the code about upgrading Arquero when we have standard library versioning, and I suggest we update the `insertArqueroTable` function then. (We may also want the current implementation around for older notebooks.)